### PR TITLE
added jumpsearch

### PR DESCRIPTION
**Pull request by akshay-gupta123**

### CODE REVIEW

File: Algorithm/jumpsearch.cpp [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    return -1; 
+} 
+  
+// Driver program to test function 
+int main() 
+{ 
+    int n;
+    cout<<"enter no. of elements in array";
+    cin>>n;
+    int arr[n];
```
Comment (huihut) [inline Algorithm/jumpsearch.cpp]: `n` should be a constant

Comment (akshay-gupta123) [reply Algorithm/jumpsearch.cpp]: we can take n to be a variable to give user the independence of no. of elements

**Comment by akshay-gupta123** [comment]
why it not get merged till now?

**Comment by huihut** [comment]
@akshay-gupta123  It can't be compiled successfully, please modify it (create heap memory with `new/delete`), or close the issue.

File: Algorithm/jumpsearch.cpp (hunk 1 above)
Comment (huihut) [reply Algorithm/jumpsearch.cpp]: You can create heap memory with `new/delete`

**Comment by huihut** [comment]
No reply for too long, so close the PR.

**Pull request closed without merging**